.circleci/cimodel/data/binary_build_definitions.py

```diff
@@ -116,6 +105,19 @@ def gen_workflow_job(self, phase, upload_phase_dependency=None, nightly=False):
         job_name = "_".join([self.get_name_prefix(), os_name, phase])
         return {job_name : job_def}
 
+def gen_binary_filter_branch(branch):
```

Comment:
Can you extend https://github.com/pytorch/pytorch/blob/df2d19723a64304aa0ca5b6887c649e25b5c2751/.circleci/cimodel/data/simple/util/branch_filters.py#L8
(by adding optional `tags` member) rather than create another inline copy here?

Reply:
done